renku/cli/exception_handler.py

```diff
@@ -68,11 +68,10 @@
 
 _BUG = click.style("Ahhhhhhhh! You have found a bug. 🐞\n\n", fg="red", bold=True)
 
-HAS_SENTRY = None
-SENTRY_DSN = os.getenv("SENTRY_DSN")
+HAS_SENTRY = os.getenv("SENTRY_ENABLED").lower() == "true"
```

Comment:
```suggestion
HAS_SENTRY = os.getenv("SENTRY_ENABLED", default="false").lower() == "true"
```
otherwise this fails if `SENTRY_ENABLED` is None (`None.lower()` doesn't work).

Pay attention, this happened here: https://github.com/SwissDataScienceCenter/renku-python/runs/5068558729?check_suite_focus=true#step:10:76 where building the cheatsheet didn't fully work because of this exception. Since the cheatsheet changed due to this bug, it was updated, so you should undo that change/commit (I think it'll automatically fix that once the bug is fixed, making a new cheatsheet commit?)

While you're at it, could you change https://github.com/SwissDataScienceCenter/renku-python/blob/develop/.github/workflows/test_deploy.yml#L160 to fail on warnings? I'm a bit surprised that sphinx build actually passed here...

Reply:
I added the `-W` flag to make it fail on warnings, and it seems to work.
Hopefully, that's only on the last-1 run of the actions as soon as you read this :grin: 

![Screenshot_20220208_174039](https://user-images.githubusercontent.com/43481553/153034805-958cd0f5-b3d2-4123-bc68-9cf77eabeafd.png)